[blog] Blogpost for upcoming price changes for MUI X (#47748)
mui-org/material-ui: docs/pages/blog/2026-MUI-X-price-changes.md

---
title: 'Upcoming Changes to MUI X Pricing and Licensing in 2026'

> description: Starting in 2026, MUI X pricing and licensing will be updated to reflect product value and support continued investment in the ecosystem.

Reviewer: ```suggestion
description: Starting in 2026, MUI X pricing and licensing is being updated to reflect product value and support continued investment in the ecosystem.
```
